src/editor/Collaboration.js

```diff
@@ -22,7 +22,7 @@ define(function (require, exports, module) {
             // immediately ask for camera access
             autoRequestMedia: false,
             // TODO : Shift this to config.
-            url: "localhost:8888"
+            url: options.collaborationUrl
```

Comment:
let's assert in the connect function before we create the webrtc object that the url exists since it is required.
```js
if(!options.serverUrl) {
    console.error(new Error("A WebRTC server url must be provided to enable collaboration."));
    return;
}
```